generator/sbpg/targets/resources/SbpTemplate.hs

```diff
@@ -5,35 +5,50 @@
 -- Maintainer:  Mark Fine <dev@swiftnav.com>
 -- Stability:   experimental
 -- Portability: portable
+--
+--
+
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
what about `OverloadedStrings` in the cabal library section's `default-extensions`?